templates/brand-store/store.html

```diff
@@ -29,8 +29,10 @@
       <div class="row">
         <h3>Installing apps</h3>
         <p>To configure a snap enabled system to use {{ webapp_config['STORE_QUERY'] }}:</p>
-        <pre><code>echo "UBUNTU_STORE_ID={{ webapp_config['STORE_QUERY'] }}" | sudo tee -a /etc/environment
```

Comment:
I will keep it as code snippet component just because we may add additional features to it. Also the base `<pre><code>` has no `.is-wrapped` feature, even though is not required here... at least not now.